using_images/s2i_images/ruby.adoc

```diff
@@ -96,6 +96,12 @@ and ruby gem installation. The application assets are only compiled if
 Asset compilation only happens when the application runs in a production
 environment. Therefore, you can use this variable when assets have already been
 compiled.
+
+|`*PUMA_MIN_THREADS*`, `*PUMA_MAX_THREADS*`
+|Indicates the minimum and maximum threads that will be available in https://github.com/puma/puma[Puma]'s thread pool.
+
+|`*PUMA_WORKERS*`
+|Indicates the number of worker processes that will be launched. See documentation on Puma's https://github.com/puma/puma#clustered-mode[clustered mode]. If not explicitly set, the default behavior sets `*PUMA_WORKERS*` to an appropriate value for the memory available to the container and the number of cores on the host.
```

Comment:
How is clustered mode related? Are worker processes always launched in clustered mode?